client/utilities/pricingConfig/digisubDiscountPricing.ts

```diff
@@ -0,0 +1,105 @@
+import { isOneOf } from '@guardian/libs';
+import type { PaidSubscriptionPlan } from '@/shared/productResponse';
+import { type CurrencyIso, CurrencyIsos } from '../currencyIso';
+
+/* 
+	Pricing for 2023 Digisub Price Rise and Save Journey
+*/
+const billingPeriods = ['month', 'quarter', 'year'] as const;
+type BillingPeriod = typeof billingPeriods[number];
+
+const oldDigisubPricePerCurrency: Record<
+	CurrencyIso,
+	Record<BillingPeriod, number>
+> = {
+	GBP: {
+		month: 11.99,
+		quarter: 35.95,
+		year: 119,
+	},
+	USD: {
+		month: 19.99,
+		quarter: 59.95,
+		year: 199,
+	},
+	EUR: {
+		month: 14.99,
+		quarter: 44.95,
+		year: 149,
+	},
+	CAD: {
+		month: 21.95,
+		quarter: 65.85,
+		year: 219,
+	},
+	AUD: {
+		month: 21.5,
+		quarter: 64.5,
+		year: 215,
+	},
+	NZD: {
+		month: 21.5,
+		quarter: 64.5,
+		year: 215,
+	},
+};
+
+const newDigisubPricePerCurrency: Record<
+	CurrencyIso,
+	Record<BillingPeriod, number>
+> = {
+	GBP: {
+		month: 14.99,
+		quarter: 44.94,
+		year: 149,
+	},
+	USD: {
+		month: 24.99,
+		quarter: 74.94,
+		year: 249,
+	},
+	EUR: {
+		month: 18.99,
+		quarter: 56.19,
+		year: 187,
+	},
+	CAD: {
+		month: 27.44,
+		quarter: 82.31,
+		year: 274,
+	},
+	AUD: {
+		month: 26.99,
+		quarter: 79.99,
+		year: 269.99,
+	},
+	NZD: {
+		month: 26.99,
+		quarter: 79.99,
+		year: 269.99,
+	},
+};
+
+function getDigisubPrice(
+	{ currency, billingPeriod }: PaidSubscriptionPlan,
+	pricePerCurrency: Record<CurrencyIso, Record<BillingPeriod, number>>,
+): number {
+	if (!isOneOf(billingPeriods)(billingPeriod)) {
+		throw new Error('Unsupported digisub billing period');
+	}
+
+	// Use USD for International pricing as fallback
```

Comment:
Where has this requirement come from?

Reply:
![image](https://github.com/guardian/manage-frontend/assets/114918544/78d692c2-4169-4cbb-9a9a-fa5f3567b888)
let's confirm but based on the checkout